test/api/importcsv/api_test.go

```diff
@@ -0,0 +1,114 @@
+package importcsv
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+func TestImportCSV(t *testing.T) {
+	for _, r := range TestCases {
+		t.Run(r.FileName, func(t *testing.T) {
+
+			// Upload CSV.
+			inputData := make(map[string]interface{})
+			err := Client.FilePost("/importsummaries/upload", r.FileName, &inputData)
+			if err != nil {
```

Comment:
Minor note, the same error check would do assert.Should(), but no problem.

Reply:
Will update it to assert.Should() to make the tests similar in nature